package.json

```diff
@@ -14,7 +14,7 @@
     "mongodb": "docker run --rm -p 27017:27017 mongo",
     "seed": "node -r esm tools/seeder/seeder.js",
     "lint": "eslint --ext .js,.vue --ignore-path .gitignore .",
-    "test": "jest"
+    "test": "jest --detectOpenHandles"
```

Comment:
Only need this for debugging, remove the --detectOpenHandles

Reply:
Done!